ocp_resources/tekton_task_run.py

```diff
@@ -0,0 +1,62 @@
+# API reference: https://tekton.dev/docs/tasks/taskruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class TaskRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        name=None,
```

Comment:
You should get here only what TasRun needed, all others should be passed as **kwargs to super()

Reply:
ack